code/game/objects/items/weapons/grenades/emgrenade.dm

```diff
@@ -4,6 +4,7 @@
 	icon_state = "emp"
 	item_state = "empgrenade"
 	origin_tech = list(TECH_MATERIAL = 2, TECH_MAGNET = 3)
+	matter = list(MATERIAL_STEEL = 3, MATERIAL_IRON = 2, MATERIAL_URANIUM = 2)
```

Comment:
I remember these could be ordered rather cheaply from the guild, so maybe no uranium.

Reply:
Actually, it should contain the chemical price of EMP. There is an EMP effect in the chem recipes, I recall it costs iron and some other reagent.

Reply:
Probably wouldn't be worth recycling if it just contained some chemicals you could make in a dispenser for free though.

Reply:
I'll add uranium reagent and iron material to it then.